### PR TITLE
doc: improve docstring for tuner.best_estimator()

### PR DESCRIPTION
**Pull request by knakad**
doc: improve docstring for tuner.best_estimator()

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [X] I have read the [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md) doc
- [X] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/sagemaker-python-sdk/blob/master/CONTRIBUTING.md#committing-your-change)
- [X] I have used the regional endpoint when creating S3 and/or STS clients (if appropriate)
- [X] I have updated any necessary documentation, including [READMEs](https://github.com/aws/sagemaker-python-sdk/blob/master/README.rst) and [API docs](https://github.com/aws/sagemaker-python-sdk/tree/master/doc) (if appropriate)

#### Tests

- [X] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [X] I have checked that my tests are not configured for a specific region or account (if appropriate)
- [X] I have used [`unique_name_from_base`](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/utils.py#L77) to create resource names in integ tests (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/sagemaker/tuner.py [hunk 1]
```diff
@@ -760,6 +760,19 @@ def best_estimator(self, best_training_job=None):
...
         object.
 
+        Args:
+            best_training_job (dict): Dictionary containing "TrainingJobName" and
+                "TrainingJobDefinitionName".
+                Example:
+                {
+                "TrainingJobName": "my_training_job_name",
+                "TrainingJobDefinitionName": "my_training_job_definition_name"
+                }
```
Comment (laurenyu) [inline src/sagemaker/tuner.py]: have you checked how this formats when rendered? My hunch is that you need more newlines and indentation.

Comment (knakad) [reply src/sagemaker/tuner.py]: Fixed! Good callout.

Comment (knakad) [reply src/sagemaker/tuner.py]: image

Comment (knakad) [reply src/sagemaker/tuner.py]: 7th time's the charm
image

**Pull request merged**